.github/workflows/build.yaml

```diff
@@ -232,35 +232,31 @@ jobs:
           retention-days: 1
 
   # See https://github.com/flatpak/flatpak-github-actionsfor examples
+  # It also contains instructions on how to compile for other architectures via QEMU
   flatpak:
     name: Flatpak
-    runs-on: ubuntu-latest
-    container:
-      image: bilelmoussaoui/flatpak-github-actions:gnome-45
-      options: --privileged
     strategy:
       matrix:
-        arch: [x86_64, aarch64]
+        variant:
+          - arch: x86_64
+            runner: ubuntu-latest
+          - arch: aarch64
+            runner: ARM64
       # Don't fail the whole workflow if one architecture fails
       fail-fast: false
+    runs-on: ${{ matrix.variant.runner }}
+    container:
+      # BUG https://github.com/flatpak/flatpak-github-actions/pull/179
+      # image: bilelmoussaoui/flatpak-github-actions:gnome-45
+      image: ghcr.io/darkdragon-001/flatpak-github-actions:gnome-45
```

Comment:
Until flatpak/flatpak-github-actions#179 is resolved, you probably want to copy over the packages to your own package registry and modify this line accordingly:

1. [Authenticate to the GitHub Docker registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-with-a-personal-access-token-classic)
2. [Copy via crane](https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane_copy.md):
   ```
   curl -sL "https://github.com/google/go-containerregistry/releases/latest/download/go-containerregistry_Linux_x86_64.tar.gz" | tar xz crane
   ./crane copy ghcr.io/darkdragon-001/flatpak-github-actions:gnome-45 ghcr.io/nanu-c/flatpak-github-actions:gnome-45
   ```